cmake/cmaize/cmaize.cmake

```diff
@@ -38,3 +38,11 @@ set(CMAKE_INSTALL_RPATH_USE_LINK_PATH TRUE)
 # endif("${isSystemDir}" STREQUAL "-1")
 
 list(APPEND CMAKE_INSTALL_RPATH "$ORIGIN")
+
+# possibly a hack, but allows the test exes to find the dlls
+# if (WIN32)
+#     set(CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS ON)
+#     set(CMAKE_ARCHIVE_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/lib)
+#     set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/lib)
+#     set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/bin)
+# endif (WIN32)
```

Comment:
Is this still needed or should it be deleted
```suggestion
```

Reply:
I left it mostly as a note. I can remove them, but I figured leave them as reference for later attempts at a more complete windows solution. @zachcran and I talked about it and didn't want to add these global settings for now. But someone else could use these options as a holdover for now if they so choose.

Reply:
Gotcha. Leave them in then.